spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -715,6 +717,60 @@
         end
       end
 
+      describe 'fileLinks' do
```

Comment:
There is a spec helper to enable the feature: https://github.com/opf/openproject/blob/dev/modules/storages/spec/support/enable_storages_module.rb#L44-L47

Reply:
for some reason that spec helper didn't work, while doing it manually worked. Does it have something to do with the describe being not at root level? every other usage of that helper are on describes on root level. I can have another look, though.

Reply:
spec helper no longer needed